src/ParquetMsg.chpl

```diff
@@ -1147,18 +1168,22 @@ module ParquetMsg {
     select dataType {
       when DType.Int64 {
         var e = toSymEntry(toGenSymEntry(entry), int);
-        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr, compression:int, mode, e.a);
+        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr,
+                                           compression:int, mode, e.a)[0];
       }
       when DType.UInt64 {
         var e = toSymEntry(toGenSymEntry(entry), uint);
-        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr, compression:int, mode, e.a);
+        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr,
+                                           compression:int, mode, e.a)[0];
       }
       when DType.Bool {
         var e = toSymEntry(toGenSymEntry(entry), bool);
-        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr, compression:int, mode, e.a);
+        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr,
+                                           compression:int, mode, e.a)[0];
       } when DType.Float64 {
         var e = toSymEntry(toGenSymEntry(entry), real);
-        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr, compression:int, mode, e.a);
+        warnFlag = write1DDistArrayParquet(filename, dsetname, dtypestr,
+                                           compression:int, mode, e.a)[0];
       } otherwise {
         var errorMsg = "Writing Parquet files not supported for %s type".format(msgArgs.getValueOf("dtype"));
         pqLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);
```

Comment:
I wasn't able to make the PR so that I could try it out:
```


$ make
/home/amandapotts/git/arkouda//src/registry/register_commands.bash /home/amandapotts/git/arkouda//src/registry /home/amandapotts/git/arkouda//registration-config.json /home/amandapotts/git/arkouda//ServerModules.cfg /home/amandapotts/git/arkouda//src
chpl dependencies are missing - try make test-venv or make chapel-py-venv
registered  15  commands from  4  modules
chpl --print-passes    --ccflags="-DH5_USE_110_API" --using-attribute-toolname arkouda --no-checks --no-loop-invariant-code-motion --no-fast-followers --ccflags="-O0" -smemTrack=true -smemThreshold=1048576 --no-cache-remote --interleave-memory -I/home/amandapotts/anaconda3/envs/arkouda-dev/include -L/home/amandapotts/anaconda3/envs/arkouda-dev/lib --ldflags="-Wl,-rpath,/home/amandapotts/anaconda3/envs/arkouda-dev/lib" -I/home/amandapotts/git/arkouda//dep/libiconv-install/include -L/home/amandapotts/git/arkouda//dep/libiconv-install/lib --ldflags="-Wl,-rpath,/home/amandapotts/git/arkouda//dep/libiconv-install/lib" -lhdf5 -lhdf5_hl -lzmq -liconv -lidn2 -lparquet -larrow -sarkoudaVersion="\"v2024.10.02+46.gbe369ed9f\""  /home/amandapotts/git/arkouda//src/arkouda_server.chpl -M /home/amandapotts/git/arkouda//src/compat/eq-20  /home/amandapotts/git/arkouda//src/CheckpointMsg.chpl /home/amandapotts/git/arkouda//src/IndexingMsg.chpl /home/amandapotts/git/arkouda//src/nd_support/nd_array_stamps.chpl -sMaxArrayDims=1  -sOpenReaderLockingDefault=false -sOpenWriterLockingDefault=false  /home/amandapotts/git/arkouda//src/registry/Commands.chpl -I/home/amandapotts/git/arkouda//src/parquet -o arkouda_server
Timing for driver compilation phase
--------------
                            init :   0.001 seconds
             parseAndConvertUast :   1.065 seconds
                       checkUast :   0.085 seconds
                     readExternC :   0.004 seconds
          expandExternArrayCalls :   0.005 seconds
                         cleanup :   0.045 seconds
/home/amandapotts/git/arkouda//src/RadixSortLSD.chpl:25: error: Bad identifier in 'except' clause, no known 'isSorted' defined in 'ArkoudaSortCompat'
make: *** [Makefile:397: arkouda_server] Error 1
```